routes/auth.js

```diff
@@ -15,8 +19,21 @@ router.post('/login', (req, res) => {
         res.json({ error: 'email ou mot de passe incorrect' })
       );
     }
-    const token = createToken(user);
-    res.json({ user, token });
+    connection.query('SELECT password FROM DVM_Legal_Entity WHERE email = ?', email, (error, results) => {
```

Comment:
you already have a user through User.findByEmailAndPassword (which should be renamed by the way, since it doesn't rely on password any more): no need to make another query :)